### PR TITLE
SPM-1261: Baseline update

### PR DESCRIPTION
**Pull request by Josca**
## Secure Coding Practices Checklist GitHub Link
- https://github.com/RedHatInsights/secure-coding-checklist

## Secure Coding Checklist
- [x] Input Validation
- [x] Output Encoding
- [x] Authentication and Password Management
- [x] Session Management
- [x] Access Control
- [x] Cryptographic Practices
- [x] Error Handling and Logging
- [x] Data Protection
- [x] Communication Security
- [x] System Configuration
- [x] Database Security
- [x] File Management
- [x] Memory Management
- [x] General Coding Practices

### CODE REVIEW

File: manager/controllers/baseline_update.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+
+type UpdateBaselineRequest struct {
+	Name         string          `json:"name"`
+	InventoryIDs map[string]bool `json:"inventory_ids"`
+	Config       BaselineConfig  `json:"config"`
+}
+
+type BaselineConfig struct {
+	ToTime string `json:"to_time"`
+}
```
Comment (MichaelMraka) [inline manager/controllers/baseline_update.go:27]: There's already BaselineConfig struct in database/baseline.go
```
type BaselineConfig struct {
	ToTime time.Time `json:"to_time"`
}
```
it should be probably unified.

File: manager/controllers/baseline_update.go [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+		LogAndRespError(c, err, "Database error: "+err.Error())
+		return
+	}
+
+	c.JSON(http.StatusOK, baselineID)
+}
+
+type SystemPlatform struct {
+	*models.SystemPlatform
+}
```
Comment (MichaelMraka) [inline manager/controllers/baseline_update.go:95]: I don't see this type used anywhere. Can we remove it?

**Review by MichaelMraka — CHANGES_REQUESTED** [review]

File: manager/controllers/baseline_update.go (hunk 2 above)
Comment (Josca) [reply manager/controllers/baseline_update.go]: Fixed.

File: manager/controllers/baseline_update.go (hunk 1 above)
Comment (Josca) [reply manager/controllers/baseline_update.go]: Fixed.

**Review by MichaelMraka — APPROVED** [review]

**Comment by MichaelMraka** [comment]
👍🏼

**Pull request merged**